pyodide-test-runner/pyodide_test_runner/decorator.py

```diff
@@ -1,68 +1,264 @@
 import ast
 import pickle
 import sys
-import traceback
+from ast import (
```

Comment:
Maybe we could still use them as `ast.<name>`?  it's a really short module name anyway. 

For instance, I find things like `Tuple` alone a bit confusing as it clashes with  `typing.Tuple` which people are more used seeing.

Reply:
I went this way because I was copying the output from `pprintast` which does not include the `ast.` prefix which requires pasting `ast.` into a LOT of places in complicated ast chunks. But now that it works, the other way probably does make more sense.

I may also look into using ast templating like here:
https://github.com/pyodide/pyodide/blob/8cfd9911af7754fd9a0f568de5f291aafe5ccf9a/src/py/_pyodide/_base.py#L104
 to simplify some of the ast construction.